### PR TITLE
windows/go1.11 migrate use of syscall

### PR DESCRIPTION
**Pull request by derekwbrown**
Change use of `syscall` to the windows package; the syscall functions
have been moved there for go 1.11

### CODE REVIEW

**Review by achntrl — DISMISSED** [review]
LGTM on the log side 👍

File: pkg/collector/corechecks/system/uptime_windows.go [hunk 1]
```diff
@@ -7,15 +7,15 @@
 package system
 
 import (
-	"syscall"
+	"golang.org/x/sys/windows"
```
Comment (truthbk) [inline pkg/collector/corechecks/system/uptime_windows.go]: nit: can we put these two in separate groups?

File: pkg/logs/input/file/open_file_windows.go [hunk 2]
```diff
@@ -8,27 +8,27 @@
 package file
 
 import (
+	"golang.org/x/sys/windows"
```
Comment (truthbk) [inline pkg/logs/input/file/open_file_windows.go]: nit: ditto

File: pkg/util/winutil/shutil.go [hunk 3]
```diff
@@ -57,7 +59,7 @@ func SHGetKnownFolderPath(rfid *GUID, dwFlags uint32, hToken syscall.Handle, psz
 
 // CoTaskMemFree free memory returned from SHGetKnownFolderPath
 func CoTaskMemFree(pv uintptr) {
-	syscall.Syscall(procCoTaskMemFree.Addr(), 1, uintptr(pv), 0, 0)
+	procCoTaskMemFree.Call(uintptr(pv), 0, 0)
```
Comment (truthbk) [inline pkg/util/winutil/shutil.go]: 🍰

**Review by truthbk — DISMISSED** [review]
LGTM! Just a couple of minuscule nits, you can merge after you address them, or if you're in real lazy mode just ignore them ;)

**Review by truthbk — APPROVED** [review]

**Pull request merged**